CONTRIBUTING.md

```diff
@@ -311,6 +310,14 @@ The third command ensures to install the BIDS schema code as an "editable" insta
 so that if you make changes to the schema files,
 these are automatically reflected in the sourcecode.
 
+Note that if you need to work on the some of the Python code 
+that is used to render the specification, 
+you will probably have to also run:
+
+```bash
+pip install -e tools/schemacode[render]
+```
```

Comment:
I don't think we actually need this, since `tools/schemacode/[render]` is in requirements.txt already.

Reply:
The point is the `-e`.

Reply:
Ohhh I didn't realize that. Great point!